modules/network-observability-packet-drops.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-packet-drops_{context}"]
+= Working with packet drops
+Packet loss occurs when one or more packets of network flow data fail to reach their destination. You can track these drops by enabling the `spec.agent.ebpf.features` and `spec.agent.ebpf.privileged` specifications in the `FlowCollector`.
```

Comment:
since `spec.agent.ebpf.features` is a list, it should not have enabled right?

Reply:
Right, I guess enabling isn't the right adverb here, it should be "You can track these drops by listing the `PacketDrop` parameter in the `spec.agent.ebof.features` list

Reply:
How about `by adding to the feature list section` ? Or something on those lines?